modules/nw-egress-ips-object-dual-stack.adoc

```diff
@@ -0,0 +1,148 @@
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: <object_name>
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: gateway-node-0
+  desiredState:
+    interfaces:
+      - name: vlan94
+        description: <description_of_vlan94_interface>
+        type: vlan
+        state: up
+        mtu: 9026
+        vlan:
+          base-iface: tenant-bond1
+          id: 94
+        ipv4:
+          address:
+            - ip: 192.168.118.20
+              prefix-length: 27
+          dhcp: false
+          enabled: true
+        ipv6:
+          address:
+            - ip: "2600:52:7:94::20"
+              prefix-length: 64
+          dhcp: false
+          enabled: true
+    routes:
+      config:
+        - destination: 192.168.120.0/25 <5>
+          metric: 999
+          next-hop-address: 192.168.118.1
+          next-hop-interface: vlan94
+          table-id: 254
+        - destination: 2600:52:7:120::/64 <6>
+          metric: 999
+          next-hop-address: 2600:52:7:94::1
+          next-hop-interface: vlan94
+          table-id: 254
+# ...
+----
++
+--
+where::
++
+`<name>`:: The name for the configuration object.
+`<kubernetes.io/hostname>`:: A node selector that that specifies which nodes receive the NNCP policy.
+`<ipv4.address.ip>`:: Specifies the IPv4 address from the `EgressIP` IPv4 subnet that you configured on the secondary network interface.
+`<ipv6.address.ip>`:: Specifies the IPv6 address from the EgressIP IPv6 subnet that you configured on the secondary network interface.
+`<destination>`:: Specifies the IPv4 address for pod IPv4 egress traffic.
+`<destination>`:: Specifies the IPv6 address for pod IPv6 egress traffic.
+--
+
+. Create a `NodeNetworkConfigurationPolicy` custom resource (CR) for your second node that has assigned egress IP addresses, such as `gateway-node-1`, so the node can handle traffic for the `EgressIP` object.
+
+. Create an `EgressIP` object and configure IPv4 and IPv6 addresses for the object. The following example `EgressIP` object uses selectors to identify which pods use the specified egress addresses for their outbound traffic:
++
+[source,yaml]
+----
+kind: EgressIP
+metadata:
+  name: egressip-dual-vlan94
+spec:
+  egressIPs:
+    - 192.168.118.30
+    - 2600:52:7:94::30
+  namespaceSelector:
+    matchLabels:
+      env: qa
+  podSelector:
+    matchLabels:
+      egressip: ds  
+# ...
+----
+
+. Create a `Pod` manifest file to test and validate your `EgressIP` object. The pod serves as a client workload that sends outbound traffic to verify that your `EgressIP` policy works as expected. 
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fedora-egressip-pod-ds
+  namespace: test
+  labels: <1>
+    egressip: ds
+spec:
+  containers:
+  - name: fedora-curl
+    image: registry.redhat.io/rhel9/support-tools:latest
+    command: ["/bin/bash", "-c", "sleep infinity"]
+# ...
+  nodeSelector: <2>
+    node-role.kubernetes.io/appworker: "" 
```

Comment:
Irrelevant. For the purpose of configuring dual-stack Egress IP, there is no need for Pod to be deployed on a special worker node. Keep it simple.

```suggestion

```